Shared/ViewModels/VideoPlayerModel.swift

```diff
@@ -16,10 +16,12 @@ struct Subtitle {
     var url: URL?
     var delivery: SubtitleDeliveryMethod
     var codec: String
+    var langCode: String
```

Comment:
while short, langcode may cause confusion for new contributors. please change to `languageCode`